tests/acceptance/features/apiContract/accessFileByFileId.feature

```diff
@@ -0,0 +1,92 @@
+@api
+Feature: accessing files using file id
+  As a user
+  I want to access the files using file id
+  So that I can read or make changes on the file
+
+  Background:
+    Given using spaces DAV path
+    And user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario Outline: get content of a file
+    Given user "Alice" has uploaded file with content "some data" to "/textfile.txt"
+    When user "Alice" gets the content of file "/textfile.txt" with path "<dav-path>" using WebDav API
```

Comment:
any good suggestions here?
```suggestion
    When user "Alice" gets the content of file "/textfile.txt" from fileId and path "<dav-path>" using WebDav API
```

Reply:
```suggestion
    When user "Alice" sends HTTP "GET" request to file "/textfile.txt" with path "<dav-path>"
                                                                                                                                                                                                                                      
```

Reply:
I think instead of adding this new step we can use the existing step `When user "Alice" sends HTTP method "GET" to URL "<dav-path>"`.

Reply:
would be great :+1:

Reply:
> Any good suggestions here?

re-using the already present step is great but that doesn't really answer this IMO and I'm just thinking out loud here I don't have any suggestions, maybe the feature file's name should be explanation enough that the request is done through the fileId ?